facebook_connector.py

```diff
@@ -43,34 +44,41 @@ def initialize(dump_directory="."):
         _messages_file = None
 
 
-def resolve_user_id(user_id):
+def resolve_user_id(fb_provided_identifier):
     """
     Tries to map the identifier facebook provides to the name of the user
 
     Args:
-        user_id: identifier string that is provided in the messages file
+        fb_provided_identifier: identifier string that is provided in the messages file
+            This might be the user's name or his Facebook ID
 
     Returns:
         The name of the user if it is able to find it, otherwise the input
     """
-    fb_id_pattern_match = _FB_ID_PATTERN.match(user_id)
-    if fb_id_pattern_match:
-        fb_numeric_id = fb_id_pattern_match.group(1)
-        if fb_numeric_id not in _mapped_fb_ids:
-            username = fb_numeric_id
-            try:
-                fb_user_page = urlopen("https://www.facebook.com/{}".format(fb_numeric_id))
-                fb_page_title = BeautifulSoup(fb_user_page.read()).title.string
-                possible_username = fb_page_title.split("|")[0].strip()
-                if not (username.startswith('Security Check Required') or username.startswith('Page Not Found')):
-                    username = possible_username
-            except:
-                # Wasn't able to find the user - no harm done
-                pass
-            _mapped_fb_ids[fb_numeric_id] = username
+    fb_id_pattern_match = _FB_ID_PATTERN.match(fb_provided_identifier)
+    if not fb_id_pattern_match:
+        # fb_provided_identifier is already the name of the user
```

Comment:
Unless you're 100% sure this is true, be a bit more vague.  E.g. if you were passed the empty string this wouldn't be true but this line would be hit.  I say this because during debugging people trust comments, therefore they must be accurate.  I'd instead say:

"Arg doesn't match the `_FB_ID_PATTERN` format, it therefore doesn't need to be mapped"